From bdb79f163f3e4cc1b54eb48afbfe582913c67267 Mon Sep 17 00:00:00 2001
From: Fugang Duan <b38611@freescale.com>
Date: Tue, 18 Aug 2015 11:28:08 +0800
Subject: [PATCH 0044/5242] MLK-11317-02 ARM: imx: imx6ul: add enet init for
 i.mx6ul

commit  881414844c941a64c4b8291cc8ed2cd8932d0432 from
https://source.codeaurora.org/external/imx/linux-imx.git

Add enet MAC address init.

The patch is merged from commit: 67cfceab0b77e887910f11dda13ab3a98bb0d87a

Signed-off-by: Fugang Duan <B38611@freescale.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/mach-imx/mach-imx6ul.c |    1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/mach-imx/mach-imx6ul.c b/arch/arm/mach-imx/mach-imx6ul.c
index 19f7e03..d0d99ad 100644
--- a/arch/arm/mach-imx/mach-imx6ul.c
+++ b/arch/arm/mach-imx/mach-imx6ul.c
@@ -55,6 +55,7 @@ static inline void imx6ul_enet_init(void)
 {
 	imx6ul_enet_clk_init();
 	imx6ul_enet_phy_init();
+	imx6_enet_mac_init("fsl,imx6ul-fec", "fsl,imx6ul-ocotp");
 }
 
 static void __init imx6ul_init_machine(void)
-- 
1.7.9.5

